project-static/squest/css/squest-dark.css

```diff
@@ -3,3 +3,6 @@
     background-color: #343a40 !important;
     color: #fff !important;
 }
+.dtfc-fixed-left{
```

Comment:
Actually removing this make the column working fine
![image](https://github.com/HewlettPackard/squest/assets/2573405/1e7fb840-f2fb-40c9-b030-a4a7d5c2471d)

Reply:
Nevermind, it was my cache